Library/Formula/premake5.rb

```diff
@@ -0,0 +1,18 @@
+class Premake5 < Formula
```

Comment:
Can premake5 execute premake4.lua files? Version 5 completely revamps the structure of the tool, @mikemcquaid @DomT4